ts/tests/_timeSpan.test.ts

```diff
@@ -0,0 +1,15 @@
+import {
+    expect,
+} from "chai";
+import {
+    TimeSpan,
+} from "../att/timeSpan";
+
+describe("time", () => {
+    it("Test TimeSpan 1", () => {
+        // arrange
```

Comment:
Please removed these kind of comment as well. 
The purpose I comment it last time is for education purpose only, so now it's not needed anymore.